src/compiler/scala/tools/nsc/transform/UnCurry.scala

```diff
@@ -317,8 +317,10 @@ abstract class UnCurry extends InfoTransform
           }
           else {
             def mkArray = mkArrayValue(args drop (params.length - 1), varargsElemType)
+            // if args.length < params.length the repeated argument is empty
+            def emptyVarargs = compareLengths(args, params) < 0
```

Comment:
Based on naming convention, shouldn't this be `emptyVarArgs` ?

Reply:
Could be. I count 69 occurrences of `[vV]arargs` and 24 of `[vV]arArgs`, so I'm not sure if there's much of a naming convention here.